packages/lib/src/types/index.ts

```diff
@@ -258,11 +258,12 @@ export interface OrderStatus {
  */
 export interface BrowserInfo {
     acceptHeader: string;
-    colorDepth: string;
+    colorDepth: number;
     language: string;
     javaEnabled: boolean;
-    screenHeight: string;
-    screenWidth: string;
+    javaScriptEnabled?: boolean;
```

Comment:
I don't know if there's any point adding this `javaScriptEnabled` typing without going deeper into the code to _actually_ add this property to the browser info that we collect i.e. in `browserInfo.ts`.
_And_, if we do add it then I think you'd need to checkout with our API team whether they're actually going to use it.
Although it's mandated for 3DS2 we haven't been sending it in up til now - so I don't quite know how the backend handles it - whether they generate their own `javaScriptEnabled` property?
I guess it's safe, from their side, to assume that if the frontend is even sending browserInfo, which is collected via javaScript, then javaScript must be enabled